### ISSUE TITLE
[Wiki] Add Note about GPU Drivers

### ISSUE THREAD

**Issue by rainyskye** [LunaPixelStudios/Better-MC#1357]
### Describe the Bug

I was having an issue when refocusing the game window, I would get a lag spike for 5-30s, which was weird, while in gameplay I would get 250-500FPS stable @ 4K, despite it being a fresh install of Windows 11 a couple of days ago, and me installing the latest NVIDIA driver then, Windows 11 appeared to have rolled it back (I assume to one validated for my Intel NUC).

Having a note in the Performance wiki might be good for some users. :)

(Another small thing I realized might also be helpful is ensuring that users are using their dGPU in the case of laptops with Optimus, etc.)

### Operating System

Windows 11

### Minecraft Version

1.19

### Modloader

Fabric

### Modpack Version

BMC v19.5

**Comment by Frontear** [comment]
Hi quick clarification, you're saying Windows rolled back your NVIDIA drivers to some MS-certified ones?

**Comment by rainyskye** [comment]
> Hi quick clarification, you're saying Windows rolled back your NVIDIA drivers to some MS-certified ones?

Yeah, and it was causing issues with large lag spikes when refocusing the window, so even in the case of someone not having their drivers rolled back, they might experience issues with old/outdated drivers :)

**Comment by Frontear** [comment]
Updated the performance guide according to suggestions: https://github.com/LunaPixelStudios/Better-MC/wiki/Performance-Guide
